app/src/main/java/com/amaze/filemanager/utils/Utils.java

```diff
@@ -94,15 +93,8 @@ public static void setTint(Context context, CheckBox box, int color) {
         }
     }
 
-    public static String getDate(long f) {
-        return DATE_WITH_MINUTES.format(f);
-    }
-
-    public static String getDate(long f, String year) {
-        String date = DATE_NO_MINUTES.format(f);
-        if (date.substring(date.length() - 4, date.length()).equals(year))
-            date = date.substring(0, date.length() - 6);
-        return date;
+    public static String getDate(@NonNull Context c, long f) {
+        return String.format(DATE_TIME_FORMAT, DateUtils.formatDateTime(c, f, DateUtils.FORMAT_SHOW_DATE), DateUtils.formatDateTime(c, f, DateUtils.FORMAT_SHOW_TIME));
```

Comment:
Why do you use `String.format()` here?

Reply:
To stay align with the format Amaze had been using.

https://github.com/TeamAmaze/AmazeFileManager/blob/b6e636bb9050cf866a8e470f409dcbc49ad4fbad/app/src/main/java/com/amaze/filemanager/utils/Utils.java#L61

Reply:
Ah, its ok then.